cmd/root.go

```diff
@@ -455,17 +127,17 @@ func compareIssues(ghClient github.Client, jiraClient jira.Client) error {
 	for _, ghIssue := range ghIssues {
 		found := false
 		for _, jIssue := range jiraIssues {
-			id, _ := jIssue.Fields.Unknowns.Int(fmt.Sprintf("customfield_%s", ghIDFieldID))
+			id, _ := jIssue.Fields.Unknowns.Int(config.GetFieldKey(lib.GitHubID))
```

Comment:
It's suppressing the error that the issue doesn't have the `GitHub ID` set; that can't happen since we filtered it by the `GitHub ID`. I'll add an error handler though, I suppose weird API errors are never unheard of.